# Message containing PWM value for steering servo and ESC

Header header
uint8 steering
uint8 throttle
bool override
float32 motor_rpm
